checks/automatic_dependency_update.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package checks
+
+import (
+	"strings"
+
+	"github.com/ossf/scorecard/checker"
+)
+
+const check = "Automatic-Dependency-Update"
+
+//nolint
+func init() {
+	registerCheck(check, AutomaticDependencyUpdate)
+}
+
+// AutomaticDependencyUpdate will check the repository if it contains Automatic dependency update.
+func AutomaticDependencyUpdate(c *checker.CheckRequest) checker.CheckResult {
+	result := CheckIfFileExists(check, c, fileExists)
```

Comment:
We need to later improve CheckIfFileExists so that it caches content and does not download again if it already exists. We have too many checks now that depend on files. @azeemshaikh38 fyi

Reply:
https://github.com/ossf/scorecard/issues/545

Reply:
Commented on the issue. This is definitely an improvement we should work on, but a low priority IMO.